_posts/blog/2018-11-12/2018-11-12-how-will-the-bch-hard-fork-on-nov-15-affect-openbazaar.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: "How Will The <span class='post-title-extra'>BCH Hard Fork</span> Affect OpenBazaar?"
+layout: post
+date: '2018-11-12 06:30:00 -0300'
+social_title: 'How Will The Bitcoin Cash (BCH) Hard Fork on November 15th Affect OpenBazaar?'
+social_description: 'Here is what you need to know about the Bitcoin Cash hard fork on November 15th as it relates to Openbazaar.'
+---
+
+#### What’s Happening with Bitcoin Cash?
+
+There is a [Bitcoin Cash](https://www.bitcoincash.org/) network upgrade due to complete at 4:40PM UTC on November 15th, 2018. Depending on how the upgrade is accepted by the network, the Bitcoin Cash blockchain could fork into two or more incompatible chains. In order to help OpenBazaar users prepare for the fork, here is some information about the OpenBazaar wallet that can help you decide how to handle any BCH funds that you may have stored there.
+
+#### How Will The Bitcoin Cash Upgrade Affect My OpenBazaar Funds?
+
+Users of OpenBazaar 2.2.5 and prior who have an active BCH wallet are using a local SPV-based wallet. [SPV wallets](https://bitcoin.org/en/glossary/simplified-payment-verification) are designed to follow [Nakamoto Consensus](https://blockonomi.com/nakamoto-consensus/), which means the fork that has the most cumulative proof-of-work according to the existing Bitcoin Cash difficulty rules. 
+
+The changes planned by both the ABC and SV implementations of the BCH network upgrade do not have any technical changes which are expected to interrupt your ability to send or receive funds within this wallet after the fork.
+
+Users of an OpenBazaar wallet are in complete control of their funds through sole access to their keys. The wallet maintains its keys and seed locally on your computer and we recommend you also maintain backups of the seed mnemonic in a safe location. At any time you can also import your seed into a BIP39-compatible wallet (such as [Electron Cash](https://electroncash.org/) ) to access your funds independent of your OpenBazaar installation. This also means that, in the event of a fork, you have the responsibility and choice to move to funds onto whichever side of the fork that you wish.
+
+The [OB1](https://ob1.io) team will continue to observe the network upgrade and will take further actions as necessary to ensure the safety of our users' funds at rest within the OpenBazaar wallet.
+
+#### What Should I Do Next?
+
+First, make sure to [backup your BCH wallet using these steps](https://openbazaar.zendesk.com/hc/en-us/articles/115002892791-How-do-I-backup-my-OpenBazaar-wallet-).
+
+Second, for the next few weeks, make sure you have the keys to your wallets as complications may arise. If you withdraw your coins from OpenBazaar, make sure it is to address you control with your private keys. 
```

Comment:
"make sure you have the keys to your wallets" -> "make sure you have the keys to your OpenBazaar wallet stored somewhere safe".

> make sure it is to address

an address

> with your private keys

a private key